boards/arm/sam4s_xplained/pinmux.c

```diff
@@ -0,0 +1,154 @@
+/*
```

Comment:
Since SAM4S is using now ASF and a new setup based on SAME70 this file is no longer necessary. The required pin configuration is provided by soc_pinmap.h file. pinmux.c can be deleted.